README.md

```diff
@@ -62,11 +62,35 @@ There are totally 19 workloads in HiBench. The workloads are divided into 6 cate
 
 3. Logistic Regression (lr)
 
-    This workload benchmarks Logistic Regression implemented in Spark-MLLib examples. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
+    This workload benchmarks Logistic Regression (lr) implemented in Spark-MLLib. Logistic Regreesion is realized with LBFGS. The input data set is generated by LabeledPointDataGenerator based on random balance decision tree. It contains three different kinds of data types, including categorical data, continuous data, and binary data.
```

Comment:
This workload benchmarks Logistic Regression with LBFGS optimizer (based on RDD-API).